scene/main/node.cpp

```diff
@@ -1635,6 +1636,12 @@ void Node::remove_child(Node *p_child) {
 	}
 }
 
+void Node::clear_children(bool p_include_internal) {
+	for (Object *n : get_children(p_include_internal)) {
+		((Node *)n)->queue_free();
+	}
+}
```

Comment:
`get_children` creates an additional typed array. Better to use `get_child_count` and a `for` loop directly I think. 
And even when doing it this way you could use `const Node *n :` directly or am I missing something (not sure on the const, someone with more c++ knowledge might have more insight).
Nitpick: the codebase uses `E` as name for such iterator variables.